app/javascript/src/all_casa_admin/tables.js

```diff
@@ -1,11 +1,8 @@
 /* global $ */
 
-$(() => { // JQuery's callback for the DOM loading
-  if ($('table.admin-list').length > 0) {
-    $('table.admin-list').DataTable({ searching: true, order: [[0, 'asc']] })
-  }
+import { initializeDataTable } from '../../datatable'
 
-  if ($('table.organization-list').length > 0) {
-    $('table.organization-list').DataTable({ searching: true, order: [[0, 'asc']] })
-  }
+$(() => {
+  initializeDataTable('table.admin-list')
+  initializeDataTable('table.organization-list')
```

Comment:
Do we have to worry about `table.admin-list` or `table.organization-list` not being defined? Will this through a JS error in the console on pages where these elements don’t exist?

Reply:
I don't think this code should throw error as $(selector).length check in the initializeDataTable function will evaluate to 0, and the DataTable initialization code inside the if block will not be executed.